great_expectations/core/expectation_configuration.py

```diff
@@ -585,45 +547,9 @@ def _update_dependencies_with_expectation_suite_urn(
     def _get_expectation_impl(self) -> Type[Expectation]:
         return get_expectation_impl(self.expectation_type)
 
-    @public_api
-    def validate(
```

Comment:
Why is a config object doing anything validation related? The Expectation may also not be the right place for this but we shouldn't duplicate efforts if we already have this on the domain obj.